.github/workflows/enhanced_gradients-publish.yml

```diff
@@ -3,6 +3,8 @@ name: enhanced_gradients publish
 on:
   push:
     tags: ["enhanced_gradients-v*"]
+    branches:
+      - "master"
```

Comment:
Consider unifying the style of yaml arrays

Reply:
I've kept the tags style from the other configs. I think it does no harm as long as it reasonably fits in one line though